src/constants.ts

```diff
@@ -198,6 +223,20 @@ export const USER_SELECTABLE_NETWORKS = [1, 42161, 43114];
 export const NEWEST_NETWORK_ID = 43114;
 
 export const NETWORKS: { [key: number]: INetwork } = {
+  4002: {
+    chainName: "Fantom Testnet",
+    chainId: 4002,
+    nativeCurrency: {
+      name: "Fantom",
+      symbol: "FTM",
+      decimals: 18,
+    },
+    rpcUrls: [""],
+    blockExplorerUrls: ["https://ftmscan.com/#/"],
+    image: ethereum,
+    imageAltText: "Ethereum Logo",
```

Comment:
```suggestion
    imageAltText: "Fantom Logo",
```

Reply:
@puf17640

Reply:
well for now it's not the fantom logo so I did not change it, will be changed with the logo

Reply:
ok got it